backend/de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/invoicecandidates/impl/CreateInvoiceCandidatesService.java

```diff
@@ -266,11 +269,14 @@ private NewManualInvoiceCandidate createCandidate(@NonNull final JsonCreateInvoi
 		{
 			final List<InvoiceDetailItem> invoiceDetailItems = item.getInvoiceDetailItems()
 					.stream()
-					.map(jsonDetail -> new InvoiceDetailItem(
-							jsonDetail.getSeqNo(),
-							jsonDetail.getLabel(),
-							jsonDetail.getDescription(),
-							jsonDetail.getDate()))
+					.map(jsonDetail -> InvoiceDetailItem.builder()
```

Comment:
please also set the orgId here; then you can avoid loading it when saving the item